docs/user/openstack/README.md

```diff
@@ -501,6 +501,14 @@ spec:
           availabilityZone: <optional_openstack_availability_zone>
 ```
 
+[provider-spec-definition]: https://github.com/openshift/cluster-api-provider-openstack/blob/155384b859c5b2fb5b7f11c9111d3f8e4f3066bd/pkg/apis/openstackproviderconfig/v1alpha1/types.go#L31
+
+#### Defining a New MachineSet That Uses Multiple Networks
```

Comment:
do you need the word "New"? Defining usually means making the first of something.